config/rbac/workload_manager_role.yaml

```diff
@@ -0,0 +1,86 @@
+# Permissions for a WLM (workload manager) to interact with DWS.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: workload-manager
+rules:
+- apiGroups:
+  - dws.cray.hpe.com
+  resources:
+  - computes
+  verbs:
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - dws.cray.hpe.com
+  resources:
+  - directivebreakdowns
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - dws.cray.hpe.com
+  resources:
+  - directivebreakdowns/status
+  verbs:
+  - get
+- apiGroups:
+  - dws.cray.hpe.com
+  resources:
+  - servers
+  verbs:
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - dws.cray.hpe.com
+  resources:
+  - servers/status
+  verbs:
+  - get
+- apiGroups:
+  - dws.cray.hpe.com
+  resources:
+  - storages
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - dws.cray.hpe.com
+  resources:
+  - storages/status
+  verbs:
+  - get
+- apiGroups:
+  - dws.cray.hpe.com
+  resources:
+  - systemconfigurations
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - dws.cray.hpe.com
+  resources:
+  - systemconfigurations/status
+  verbs:
+  - get
+- apiGroups:
+  - dws.cray.hpe.com
+  resources:
+  - workflows
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
James isn't doing anything with this right now, but I wonder if he'll want to look at persistentstorageinstances. It would be useful for users to see which ones exist. I'm not sure if that would be through flux or some other tool, though.

Reply:
Also, he'll want read only access to ClientMounts

Reply:
done